src/backend/distributed/master/master_create_shards.c

```diff
@@ -138,6 +139,24 @@ CreateShardsWithRoundRobinPolicy(Oid distributedTableId, int32 shardCount,
 						errmsg("replication_factor must be positive")));
 	}
 
+	/* make sure that RF=1 if the table is streaming replicated */
+	if (cacheEntry->replicationModel == REPLICATION_MODEL_STREAMING &&
+		replicationFactor > 1)
+	{
+		ereport(ERROR, (errcode(ERRCODE_INVALID_PARAMETER_VALUE),
+						errmsg("using replication factor %d with streaming "
+							   "replicated tables are not supported",
```

Comment:
-> using replication factor %d with the streaming replication model is not supported